src/components/biggive-search/readme.md

```diff
@@ -0,0 +1,3 @@
+# biggive-search
+
+Provides a text field for users to search for campaigns and/or charities.
```

Comment:
This should include the auto generated docs from `build` in order to:
1. keep diffs clean
2. stop future runs deleting your summary